doc/api/errors.md

```diff
@@ -1004,6 +1004,22 @@ Used when a callback is called more then once.
 can either be fulfilled or rejected but not both at the same time. The latter
 would be possible by calling a callback more then once.
 
+<a id="ERR_NET_INVALID_LISTEN_ARG"></a>
+### ERR_NET_INVALID_LISTEN_ARG
```

Comment:
If I'm understanding correctly, the error isn't (necessarily?) an invalid argument **type**, but an invalid argument.

If we can make `ERR_INVALID_OPT_VALUE` work instead, that would be great. Will that result in a similar difficulty? If so, then I'm OK with a new error. If we've been trying for more general errors, E_INVALID_OPTION_OBJECT or something like that might be preferable so it can be re-used elsewhere. But there's a valid argument for specific codes too, so either way is fine by me.